Library/Homebrew/Gemfile

```diff
@@ -41,6 +41,7 @@ end
 gem "activesupport"
 gem "addressable"
 gem "concurrent-ruby"
+gem "i18n"
```

Comment:
We're using this explicitly, but we were relying on it as a transitive dependency of `activesupport`, so I've made it an explicit dependency.